src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -23,11 +24,29 @@
             + "Example: " + COMMAND_WORD + " 1";
```

Comment:
Update message usage to reflect the flag.
Currently, invalid format gives:
`Invalid command format! 
delete: Deletes the person identified by the index number used in the displayed person list.
Parameters: INDEX (must be a positive integer)`